protos/ray.proto

```diff
@@ -49,6 +49,14 @@ service Scheduler {
   rpc SchedulerInfo(SchedulerInfoRequest) returns (SchedulerInfoReply);
   // Get information about tasks
   rpc TaskInfo(TaskInfoRequest) returns (TaskInfoReply);
+  // Kills the workers
+  rpc KillWorkers(KillWorkersRequest) returns (KillWorkersReply);
+}
+
+message KillWorkersRequest {
```

Comment:
The definitions of all of the messages should appear in the same order as the rpc definitions in the Scheduler service. That is, `message KillWorkersRequest` should come after `message TaskInfoReply`. The same goes for the Worker Service messages.